test/onnx/test_operators.py

```diff
@@ -681,12 +685,6 @@ def test_dropout_training(self):
         x = torch.randn(3, 4, requires_grad=True)
         self.assertONNX(lambda x: torch.max(functional.dropout(x)), x, training=torch.onnx.TrainingMode.TRAINING)
 
-    @unittest.skip("disable test until onnx submodule is updated")
```

Comment:
This is probably the reason why CI failing. I suppose they are removed by accident?